document/paragraphproperties.go

```diff
@@ -199,3 +199,186 @@ func (p ParagraphProperties) SetHangingIndent(m measurement.Distance) {
 		p.x.Ind.HangingAttr.ST_UnsignedDecimalNumber = unioffice.Uint64(uint64(m / measurement.Twips))
 	}
 }
+
+// Bold returns true if paragraph font is bold.
+func (p ParagraphProperties) Bold() bool {
+	x := p.x.RPr
+	return getBool(x.B) || getBool(x.BCs)
+}
+
+// Italic returns true if paragraph font is italic.
+func (p ParagraphProperties) Italic() bool {
+	x := p.x.RPr
+	return getBool(x.I) || getBool(x.ICs)
+}
+
+// Caps returns true if paragraph font is capitalized.
+func (p ParagraphProperties) Caps() bool {
+	return getBool(p.x.RPr.Caps)
+}
+
+// Strike returns true if paragraph is striked.
+func (p ParagraphProperties) Strike() bool {
+	return getBool(p.x.RPr.Strike)
+}
+
+// DoubleStrike returns true if paragraph is double striked.
+func (p ParagraphProperties) DoubleStrike() bool {
+	return getBool(p.x.RPr.Dstrike)
+}
+
+// Outline returns true if paragraph outline is on.
+func (p ParagraphProperties) Outline() bool {
+	return getBool(p.x.RPr.Outline)
+}
+
+// Shadow returns true if paragraph shadow is on.
+func (p ParagraphProperties) Shadow() bool {
+	return getBool(p.x.RPr.Shadow)
+}
+
+// Emboss returns true if paragraph emboss is on.
+func (p ParagraphProperties) Emboss() bool {
+	return getBool(p.x.RPr.Emboss)
+}
+
+// RightToLeft returns true if paragraph text goes from right to left.
+func (p ParagraphProperties) RightToLeft() bool {
+	return getBool(p.x.RPr.Rtl)
+}
+
+// RStyle returns the name of paragraph runs style.
+func (p ParagraphProperties) RStyle() string {
```

Comment:
What values can this take?  Is that documented somewhere? Either list or provide a reference to documentation

Reply:
It is a user-defined style which he can define here 
![image](https://user-images.githubusercontent.com/8694210/81653065-30270180-943d-11ea-8ab1-ca9633a7b9d2.png)
, so no lists are there, I pointed to URL